src/SortingTable/SortingTable.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Table, TableProps } from "../Table";
```

Comment:
Can we split the named and type import as so: 

```suggestion
import { Table } from "../Table";
import type { TableProps } from "../Table";
```

Reply:
Looks like another typescript thing - sure I will change that.

Reply:
Done